app/utils.py

```diff
@@ -101,6 +101,10 @@ def query2str(query):
     return query.statement.compile(compile_kwargs={"literal_binds": True})
 
 
+def parse_querystring(query: str) -> dict:
```

Comment:
Nits: this method is small enough so I think we can use `urllib.parse.parse_qs(query)` directly :)

Reply:
Removed it